File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        map<int, int> count;
        for (int i = 0; i<nums.size(); i++){
            count[nums[i]]++;
        }

        vector<pair<int, int>> v(count.begin(), count.end()); // copy map -> vector
        sort(v.begin(), v.end(), [](pair<int, int>& a, pair<int, int>& b){ 
            return a.second > b.second; 
            });

        vector<int> result;
        for(int i =0; i<k; i++){
            result.push_back(v[i].first);
        }
        return result;
    }
};

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        j = 0
        count = 0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[j] = nums[i]
                j +=1
            else: count +=1
        
        k = len(nums) - count
        return k


